rust/Cargo.lock

```diff
@@ -1062,15 +1319,24 @@ dependencies = [
  "winapi",
 ]
 
+[[package]]
+name = "ripemd"
+version = "0.1.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "1facec54cb5e0dc08553501fa740091086d0259ad0067e0d4103448e4cb22ed3"
+dependencies = [
+ "digest 0.10.3",
+]
+
 [[package]]
 name = "ripemd160"
```

Comment:
Ideally we would only depend on ripemd. The dependency on ripemd160 is from hdwallet-bitcoin.

Reply:
The ripemd dependency has now been removed entirely, as it's transitive.